### DISCUSSION TITLE
Global Search not working as expected.

### DISCUSSION THREAD

**Discussion by yuvrajtimalsina** [filamentphp/filament#8231] [General]
I have a model for **Setting** with the following :

```
    public $incrementing = false;

    protected $primaryKey = 'key';

    protected $fillable = [
        'key',
        'value',
        'label',
        'type',
    ];
```
In **SettingResource** I couldn't get results via Global Search : 

```
protected static ?string $recordTitleAttribute = 'label';
```
I also tried with : 
```
  public static function getGlobalSearchResultTitle(Model $record): string
    {
        return $record->label;
    }
```

Am I missing something here?

**Comment by danharrin** [comment]
Is it a simple resource with modals instead of pages?

**Reply by yuvrajtimalsina** [reply]
Yes, it is indeed.I happen to notice a note as well but I was expecting a workaround.

**Reply by danharrin** [reply]
Global search doesn't support simple resources yet as we cannot open modals on page load yet

**Reply by yuvrajtimalsina** [reply]
Thanks Dan, I will happily create Edit page then.

**Reply by danharrin** [reply]
Implemented in #9358.
